src/lib/Behat/BrowserContext/UserPreferencesContext.php

```diff
@@ -29,4 +37,14 @@ public function iChangePassword($oldPassword, $newPassword): void
         $this->changePasswordPage->setNewPassword($newPassword);
         $this->changePasswordPage->setConfirmPassword($newPassword);
     }
+
+    /**
+     * @When I set Autosave draft option to :autosaveDraftValue value
```

Comment:
I'd prefer a step named: `I disable autosave`

Reply:
But what if hypothetically there will be a need to add step like `I enable autosave` ? All im saying is that current step could possibly cover enablig and disabling of autosave :)